apps/readest-app/src/components/settings/SettingsDialog.tsx

```diff
@@ -74,6 +82,11 @@ const SettingsDialog: React.FC<{ bookKey: string }> = ({ bookKey }) => {
       icon: IoAccessibilityOutline,
       label: _('Custom'),
     },
+    {
```

Comment:
Please reimplement this. `Replacement` should not be a separate tab in the Settings dialog. Please keep all `Replacement` configuration accessible in the sidebar book menu.